test/framework/helpers.go

```diff
@@ -261,7 +261,7 @@ func getMetricValue(b []byte, metricName string) (float64, error) {
 		}
 
 		seriesLabels := labels.Labels{}
-		parser.Metric(&seriesLabels)
+		parser.Labels(&seriesLabels)
```

Comment:
https://github.com/prometheus-operator/prometheus-operator/actions/runs/14485091657/job/40629135389?pr=7462

```
  Error: test/e2e/alertmanager_test.go:49:16: could not import github.com/prometheus-operator/prometheus-operator/test/framework (-: # github.com/prometheus-operator/prometheus-operator/test/framework
  Error: test/framework/helpers.go:264:10: parser.Metric undefined (type textparse.Parser has no field or method Metric)) (typecheck)
  	testFramework "github.com/prometheus-operator/prometheus-operator/test/framework"
```

Breaking change due to https://github.com/prometheus/prometheus/commit/00b69efabb3cd3e213a2a34939f745d7156f7d47